test/UnitTests/MSTestAdapter.PlatformServices.UnitTests/TestableImplementations/TestablePlatformServiceProvider.cs

```diff
@@ -37,7 +37,7 @@ internal class TestablePlatformServiceProvider : IPlatformServiceProvider
 
     #endregion
 
-    public ITestSource TestSource => MockTestSourceValidator.Object;
+    public ITestSourceHandler TestSource => MockTestSourceValidator.Object;
```

Comment:
As `TestSource` property was removed from the interface, I guess this property is no longer needed?

Maybe even `MockTestSourceValidator` should no longer belong here, and individual tests should just manually create a mock.